NGitLab.Tests/EnvironmentsTests.cs

```diff
@@ -84,15 +85,19 @@ public async Task Edit()
         // Validate newEnvNameToEdit is present
         Assert.That(envClient.All.FirstOrDefault(e => string.Equals(e.Name, newEnvNameToEdit, StringComparison.Ordinal)), Is.Not.Null);
 
+        // Handling for old version 15.x
+        var isVersion15 = context.IsGitLabVersionInRange(VersionRange.Parse("[15.0,16.0)"), out _);
+
         // Edit and check return value
         env = envClient.Edit(initialEnvId, newEnvNameUpdated, newEnvNameExternalUrlUpdated);
-        Assert.That(env.Name, Is.EqualTo(newEnvNameUpdated).IgnoreCase);
+        if (isVersion15) { Assert.That(env.Name, Is.EqualTo(newEnvNameUpdated).IgnoreCase); }
```

Comment:
```suggestion

        if (isVersion15)
        { 
            Assert.That(env.Name, Is.EqualTo(newEnvNameUpdated).IgnoreCase);
        }

```

Not a fan on one one line conditions, can you indent the others too?

Reply:
sure

Reply:
Done